docs/src/app/components/pages/components/Chip/Page.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Title from 'react-title-component';
+
+import CodeExample from '../../../CodeExample';
+import PropTypeDescription from '../../../PropTypeDescription';
+import MarkdownElement from '../../../MarkdownElement';
+
+import chipReadmeText from './README';
+import ChipExampleSimple from './ExampleSimple';
+import chipExampleSimpleCode from '!raw!./ExampleSimple';
+import chipCode from '!raw!material-ui/lib/Chip/Chip';
+import enhancedButtonCode from '!raw!material-ui/lib/internal/EnhancedButton';
+
+const description = 'Examples of Chips using an image, [Font Icon](/#/components/font-icon), ' +
```

Comment:
Looking at the docs for that rule, it does only gives examples of single-line concatenation, but I just tested, and that passes too. Odd!